test/runtests.jl

```diff
@@ -1,10 +1,42 @@
 # Check that package loads
 using StatsKit
 
-using Pkg
+using Test
 
 # TODO: run all tests in the same process after 'using StatsKit'
 #       to check that they pass when all packages are loaded at the same time
-Pkg.test(["Bootstrap", "CategoricalArrays", "Clustering", "CSV", "DataFrames",
-          "Distances", "Distributions", "GLM", "HypothesisTests", "KernelDensity",
-          "Loess", "MultivariateStats", "MixedModels", "StatsBase", "ShiftedArrays", "TimeSeries"])
+#       This is not possible currently as Pkg.test starts a new environment
+
+# Check that reexported packages do not export different objects under the same name
+objects = Dict{Any, Symbol}()
+for m in [Bootstrap, CategoricalArrays, Clustering, CSV, DataFrames,
+          Distances, Distributions, GLM, HypothesisTests, KernelDensity,
+          Loess, MultivariateStats, MixedModels, StatsBase, ShiftedArrays, TimeSeries]
+    for name in names(m)
+        # Work around undefined exports:
+        # JuliaStats/StatsBase#663,
+        # JuliaStats/Clustering.j#210,
+        # JuliaStats/GLM.jl#405,
+        # JuliaStats/StatsModels.jl#218,
+        # JuliaStats/MultivariateStats.jl#142,
+        # JuliaArrays/ShiftedArrays.jl#42,
+        # JuliaStats/TimeSeries.jl#487
```

Comment:
That PR has been merged already JuliaStats/TimeSeries.jl#487
```suggestion
```

Reply:
Indeed. I've pushed a commit to remove a few functions which have been fixed since then.

Reply:
Actually, it appears that Julia 1.0 still uses old versions of most of these packages (because of some dependencies?). Let's keep this for now, once the new LTS is out we'll probably drop support for 1.0.